src/Umbraco.Tests.AcceptanceTest/cypress/integration/Tour/backofficeTour.ts

```diff
@@ -1,49 +1,99 @@
+    runBackOfficeIntroTour(0, 'Start');
+
     //assert
     cy.get('[data-element="help-tours"]').should("be.visible");
-    //act
     cy.get('[data-element="help-tours"]').click();
-    //assert
-    cy.get('[data-element="tour-umbIntroIntroduction"] .umb-button').should("be.visible");
+    cy.get('[data-element="help-tours"] .umb-progress-circle', { timeout: 60000 }).contains('17%');
+  });
+
+  it('Backoffice introduction tour should run then rerun', () => {
     //act
-    cy.get('[data-element="tour-umbIntroIntroduction"] .umb-button').click();
-    //assert
-    cy.get('.umb-tour-step', { timeout: 60000 }).should('be.visible');
-    cy.get('.umb-tour-step__footer').should('be.visible'); 
-    cy.get('.umb-tour-step__counter').should('be.visible');
-
-    for(let i=1;i<7;i++){
-      cy.get('.umb-tour-step__counter').contains(i + '/12');
-      cy.get('.umb-tour-step__footer .umb-button').should('be.visible').click();
-    }
-    cy.umbracoGlobalUser().click()
-    cy.get('.umb-tour-step__counter').contains('8/12');
-    cy.get('.umb-tour-step__footer .umb-button').should('be.visible').click();
-    cy.get('.umb-tour-step__counter').contains('9/12');
-    cy.get('.umb-overlay-drawer__align-right .umb-button').should('be.visible').click();
-    cy.get('.umb-tour-step__counter').contains('10/12');
-    cy.umbracoGlobalHelp().click()
-
-    for(let i=11;i<13;i++){
-      cy.get('.umb-tour-step__counter').contains(i + '/12');
-      cy.get('.umb-tour-step__footer .umb-button').should('be.visible').click();
-    }
-    cy.get('.umb-tour-step__footer .umb-button').should('be.visible').click();
+    cy.umbracoGlobalHelp().should("be.visible");
+    cy.umbracoGlobalHelp().click();
+    runBackOfficeIntroTour(0, 'Start');
+    runBackOfficeIntroTour(17, 'Rerun');
 
     //assert
+    cy.get('[data-element="help-tours"]').should("be.visible");
+    cy.get('[data-element="help-tours"]').click();
+    cy.get('[data-element="help-tours"] .umb-progress-circle', { timeout: 60000 }).contains('17%');
     cy.umbracoGlobalHelp().should("be.visible");
-    cy.get('[data-element="help-tours"] .umb-progress-circle').contains('17%');
-   });
+  });
 });
+
+afterEach(() => {
+  //cleanup
+  resetTourData();
+});
+
+function resetTourData() {
+  var tourStatus =
+  {
+    "alias": "umbIntroIntroduction",
+    "completed": false,
+    "disabled": false
+  };
+
+  cy.getCookie('UMB-XSRF-TOKEN', { log: false }).then((token) => {
+    cy.request({
+      method: 'POST',
+      url: '/umbraco/backoffice/UmbracoApi/CurrentUser/PostSetUserTour',
+      followRedirect: false,
+      headers: {
+        ContentType: 'application/json',
+        'X-UMB-XSRF-TOKEN': token.value,
+      },
+      body: tourStatus,
+    }).then((resp) => {
+      return;
+    });
+  })
+}
+
+function runBackOfficeIntroTour(percentageComplete, buttonText) {
+  cy.get('[data-element="help-tours"]').should("be.visible");
+  cy.get('[data-element="help-tours"]').click();
```

Comment:
```suggestion
    cy.get('[data-element="help-tours"] .umb-help-list').first().click();
```

Seems to be necessary because there are more tours available now?

Reply:
This was a bit tricky.. Because there was 3 yours, the `cy.get('[data-element="help-tours"]').click()`; was clicking in the center of the `[data-element="help-tours"]`. This was the second tour, and the the first as expected

Reply:
Thanks @bergmania  - I will take a look later in the week, and good point re the additional tours now!

Reply:
@bergmania I've now updated the tests, there is probably room for some refactoring but I've run several times now and green for me both times. Let me know if locally this is ok for you now.

Reply:
I still have this issue. At you can see on these images it clicks on the wrong tour:
![image](https://user-images.githubusercontent.com/1561480/91128702-ed65fe00-e6a8-11ea-9fc0-aa803c14a584.png)

![image](https://user-images.githubusercontent.com/1561480/91128644-daebc480-e6a8-11ea-8785-718826fe060d.png)

Reply:
My bad, I see these extra tours are comming from starter kit - Will retest